Scripts/Modules/HapticsModule/HapticPulse.cs

```diff
@@ -0,0 +1,29 @@
+﻿#if UNITY_EDITOR
+using UnityEngine;
+
+namespace UnityEditor.Experimental.EditorVR.Core
+{
+	class HapticPulse : ScriptableObject
```

Comment:
You should be able to do:
[CreateAssetMenu(menuName = "EditorVR/HapticPulse")] (or some other name/path you like) and get rid of the CreateHapticPulse class altogether